src/axom/sidre/tests/sidre_native_layout.cpp

```diff
@@ -520,6 +521,146 @@ TEST(sidre_native_layout,import_conduit_and_verify_protocol)
   }
 }
 
+//------------------------------------------------------------------------------
+
+TEST(sidre_native_layout, basic_demo_compare)
+{
+  //
+  // create demo a datastore that includes:
+  //  groups
+  //  views:
+  //    scalar, string, array, external array, array of sliced buffer
+
+  DataStore ds;
+  Group* root = ds.getRoot();
+
+  // Note:
+  // we use int64 and float64 b/c those types persist even with
+  // json or yaml output
+
+  conduit::int64     sidre_vals_1[5] = {0,1,2,3,4};
+  conduit::float64   sidre_vals_2[6] = { 1.0, 2.0,
+                                         1.0, 2.0,
+                                         1.0, 2.0,};
+
+  Group* group1 = root->createGroup("my_scalars");
+  // two scalars
+  group1->createViewScalar<conduit::int64>("i64", 1);
+  group1->createViewScalar<conduit::float64>("f64", 10.0);
+
+  Group* group2 = root->createGroup("my_strings");
+  // two strings
+  group2->createViewString("s0", "s0 string");
+  group2->createViewString("s1", "s1 string");
+
+  // one basic array
+  Group* group3 = root->createGroup("my_arrays");
+  View* a5_std_view = group3->createViewAndAllocate("a5_i64_std",
```

Comment:
is `std` for standard? as opposed to `ext` for external?

Reply:
yes, wanted names tried to reflect the cases. could be better

Reply:
maybe `int_view` ( vs. `ext_view`) ?

Reply:
`int` == `internal` or `integer` :-)

I think i can just drop std?